manifests/site.pp

```diff
@@ -51,7 +58,7 @@
     include role::roundcubemail
 }
 
-node /^mem[12]\.miraheze\.org$/ {
+node /^mem([12]|1([02]1)\.miraheze\.org$/ {
```

Comment:
Missing a closing parentheses, so this is invalid regex. `/^mem([12]|1([02]1))\.miraheze\.org$/`